tools/release.js

```diff
@@ -7,6 +7,8 @@ const { exec } = require("child_process");
 const loading = require("loading-indicator");
 const MAIN_BRANCH = "develop";
 const DEBUG = false;
+const vnu = require("vnu-jar");
+const tmp = require("tmp");
```

Comment:
Can we avoid adding this dependency (and its dependency) for something Node.js allows easily?

Reply:
```js
const tempFile = path.join(os.tmpdir(), "index.html");
```